<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>481</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>70</y>
      <width>391</width>
      <height>160</height>
     </rect>
    </property>
    <property name="title">
     <string>ステージ移動</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>19</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X軸</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>69</y>
       <width>19</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y軸</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>98</y>
       <width>19</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z軸</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>127</y>
       <width>28</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ピエゾ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>22</y>
       <width>29</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[μm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>22</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[μm/s]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_lineEdit">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>41</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_lineEdit">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="z_lineEdit">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>99</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="p_lineEdit">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>128</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_ms_lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>41</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_ms_lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="z_ms_lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>99</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="p_ms_lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>128</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="x_pushButton">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>移動</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y_pushButton">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>69</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>移動</string>
     </property>
    </widget>
    <widget class="QPushButton" name="z_pushButton">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>98</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>移動</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_pushButton">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>127</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>移動</string>
     </property>
    </widget>
    <widget class="QPushButton" name="x0_pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>原点移動</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y0_pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>69</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>原点移動</string>
     </property>
    </widget>
    <widget class="QPushButton" name="z0_pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>98</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>原点移動</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p0_pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>127</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>原点移動</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>391</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>造形条件設定</string>
    </property>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>22</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>データ形式 ：</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_xy">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>22</y>
       <width>37</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XY</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_xz">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>22</y>
       <width>37</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XZ</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_yz">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>22</y>
       <width>37</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>YZ</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_xyz">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>22</y>
       <width>44</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XYZ</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>361</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>積層造形</string>
     </property>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>81</y>
        <width>34</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>開始 ：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>81</y>
        <width>34</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>終了 ：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="start_spinBox">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>81</y>
        <width>38</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="end_spinBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>81</y>
        <width>38</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>63</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>間隔[μm] ：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="interval_lineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>81</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>造形フォルダ ：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="folder_lineEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="folder_refer_pushButton">
         <property name="text">
          <string>参照</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>層データ　　 ：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="layer_lineEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="layer_refer_pushButton">
         <property name="text">
          <string>参照</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="data_reverse_checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>断面データ正負反転</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="position_return_checkBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>開始位置復帰</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>361</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>XYZ造形</string>
     </property>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>造形フォルダ ：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xyz_folder_lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="xyz_refer_pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>参照</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="reverse_checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>75</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>逆造形</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>造形方式</string>
     </property>
     <widget class="QRadioButton" name="galvano_radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ガルバノミラー</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="stage_radioButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ステージ</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>361</y>
       <width>99</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>造形速度[μm/s] ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>387</y>
       <width>99</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>造形速度[μm/s] ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>413</y>
       <width>99</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>造形速度[μm/s] ：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="v1_lineEdit">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>361</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="v2_lineEdit">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>387</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="v3_lineEdit">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>413</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>361</y>
       <width>22</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>層 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>387</y>
       <width>22</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>層 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_52">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>413</y>
       <width>22</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>層 ：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l1_start_lineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>361</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="l2_start_lineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>387</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="l3_start_lineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>413</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="renew_pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>データ更新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>造形開始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>造形停止？</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l1_end_lineEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>361</y>
       <width>39</width>
       <height>19</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="l2_end_lineEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>386</y>
       <width>39</width>
       <height>19</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="l3_end_lineEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>411</y>
       <width>39</width>
       <height>19</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>291</x>
       <y>363</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>～</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>291</x>
       <y>388</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>～</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>291</x>
       <y>413</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>～</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>ステージ位置</string>
    </property>
    <widget class="QLabel" name="Position_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>770</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>造形データの大きさ</string>
    </property>
    <widget class="QLabel" name="Size_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>770</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>造形時間目安</string>
    </property>
    <widget class="QLabel" name="Time_label">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>20</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_12">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>シャッター開閉</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>21</y>
       <width>331</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="open_pushButton">
        <property name="text">
         <string>OPEN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="close_pushButton">
        <property name="text">
         <string>CLOSE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>普及機アプリケーション</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
